spotbugs/src/test/java/edu/umd/cs/findbugs/util/ClassNameTest.java

```diff
@@ -65,4 +64,21 @@ public void testGetPrimitiveType() {
     public void testExtractClassNameBad() {
         ClassName.extractClassName("L[Ljava/lang/Integer;");
     }
+
+    @Test
+    public void testMatchedPrefix() {
```

Comment:
It might be worth considering a parameterised test for this - although JUnit doesn't have as much support for that as TestNG does. Still, iterating through a collection of test cases might be more readable and maintainable than having a repeated line for each one.